lib/js/components/DatePickers/DateRangePicker/DateRangePicker.vue

```diff
@@ -134,6 +135,10 @@ export default defineComponent({
 			type: String,
 			default: '',
 		},
+		createInstanceOnToggle: {
```

Comment:
@pioziol123 do you have an idea for a better name?

Reply:
What you think about making create on toggle the only behavior? It would simplify code a lot. Maybe we should even destroy picker when it's not used?